src/Makefile

```diff
@@ -94,6 +94,13 @@ ifeq ($(MALLOC),jemalloc)
 	FINAL_LIBS+= ../deps/jemalloc/lib/libjemalloc.a -ldl
 endif
 
+ifeq ($(USE_NVML),yes)
+	DEPENDENCY_TARGETS+= nvml
+	FINAL_CFLAGS+= -I../deps/nvml/src/include -DUSE_NVML
+	FINAL_LIBS+= ../deps/nvml/src/nondebug/libpmem.a ../deps/nvml/src/nondebug/libpmemlog.a ../deps/nvml/src/nondebug/libpmem.a -luuid
```

Comment:
So, we always link to *nondebug* version of NVML? What if user needs to build a debug version of Redis? Shouldn't we use *debug* version of *libpmemlog* in such case?